XLForm/XL/Descriptors/XLFormRowDescriptor.m

```diff
@@ -125,8 +125,13 @@ -(XLFormBaseCell *)cellForFormController:(XLFormViewController * __unused)formCo
                 NSString *bundlePath = [[[NSBundle mainBundle] resourcePath] stringByAppendingPathComponent:folderName];
                 bundle = [NSBundle bundleWithPath:bundlePath];
             } else {
-                bundle = [NSBundle bundleForClass:NSClassFromString(cellClass)];
-                cellResource = cellClassString;
+              Class cellClassActual = NSClassFromString(cellClassString);
+              if (cellClassActual == nil) {
+                // On XCode10.1 for iOS11 the module prefix is not provided so provide it here if it's missing
+                cellClassActual = NSClassFromString([NSString stringWithFormat: @"%@.%@", [formController moduleName], cellClassString]);
```

Comment:
What if `[formController moduleName]` is nil?

Reply:
If `moduleName` is nil the app crashes. I've added a NSAssert saying it needs to be set.